### ISSUE TITLE
Damage Tint not show in 1.17

### ISSUE THREAD

**Issue by Herobrine2143** [DeadlyMC/damage-tint#8]
I've tried to go in single player and the mod doesn't tint my screen when I'm close to dying. Neither when I take damage.

Fabric 1.17.1; 1.1.0 version of the mod

**Comment by DeadlyMC** [comment]
Are you running any other mods?

**Comment by Herobrine2143** [comment]
Yes, but I already tested if it was that, and I only enabled Fabric API and Mod Menu enabled, and it still didn't work

**Comment by DeadlyMC** [comment]
Do you have any resourcepacks?

**Comment by DeadlyMC** [comment]
Could you also send me your video settings

**Comment by Herobrine2143** [comment]
I'm using Sodium, so my settings are pretty much the same as the default just changing it to Fabulous.

![image](https://user-images.githubusercontent.com/22053132/131220812-b2d19def-2f6a-476f-a61f-adf2f6bf4101.png)

![image](https://user-images.githubusercontent.com/22053132/131220802-de24cf2b-ef29-4511-8d9c-1bfa0d226627.png)

![image](https://user-images.githubusercontent.com/22053132/131220821-6af4a846-e616-4f43-a99e-690fbb91dc22.png)

Here's the Vanilla UI for the video settings

![image](https://user-images.githubusercontent.com/22053132/131220846-e07becf1-5715-4e0c-83ca-40c2eaab457e.png)

and checking with the Resource Packs, it seems is because of the resource packs. I disabled every single resource pack and it works now, Apparently the cause is the Texture Pack called **New Default+** the rest of the packs don't make it disappear.

**Comment by DeadlyMC** [comment]
What does New Default+ do? I haven't heard of that

**Comment by Herobrine2143** [comment]
I'm not certainly sure since it doesn't state it's full features. You could see it here [https://www.curseforge.com/minecraft/texture-packs/newdefaultplus](url)

All I know so far is that it gives you the ability to change the tools, armor, weapons' texture by renaming them, basically using Optifine's features but that doesn't concern us since I'm not using Optifine, and will work fine without it. All features that uses Optifine will be ignored, those features will just be not visible but the rest of the texture pack will be run.

It also changes light mapping, mob re-skin, as well as all item changes and gives some blocks a 3D look like the ladder.

it probably changes the vignette which is what you use? but then again I use your mod without vignette on and still works (Not using NewDefault+ in this instance.). But I'm not sure.

**Comment by DeadlyMC** [comment]
Its definitely the vignette, i will have a look
